### ISSUE TITLE
Should ModelAdapter displays the `Status` on the result of get command

### ISSUE THREAD

**Issue by brosoul** [vllm-project/aibrix#238]
### 🚀 Feature Description and Motivation

ModelAdapter, like a Pod, displays the status  on the result of get command.
Displaying the status in this way will be more user-friendly.

[image](https://github.com/user-attachments/assets/981696c4-8745-437a-8c1a-6ca388f1fa8c)

**Comment by newhans** [comment]
> [@brosoul](https://github.com/brosoul) I am trying to understand the question. you mean implementing a plugin to show the status? I think that's reasonable, this requires some kubectl plugin I think.
> 
> ```
> kubectl get modeladapter
> xxxx
> ```
> 
> this can be definitely enriched.

Do you agree that simply modifying the CRD to include 'additionalPrinterColumns' can achieve this functionality? If so, could you assign this issue to me? @Jeffwan

**Comment by yaojiejia** [comment]
Hi @Jeffwan is this issue still valid? if so can i work on this?

**Comment by Jeffwan** [comment]
@yaojiejia yes, please pick it up. I remember we added some fields but may still lack of some, please try it out.
